app/assets/stylesheets/db/controllers/edit-requests.scss

```diff
@@ -0,0 +1,44 @@
+body.db-edit_requests-index {
+  table {
+    td.title-user {
+      .title {
+        margin-bottom: 3px;
+      }
+
+      .user {
+        a {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 3, but was 5